src/main/java/lotto/domain/PurchaseResult.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PurchaseResult {
+
+    private final Lottos manualLottos;
+    private final Lottos automaticLottos;
+
+    public PurchaseResult(Lottos manualLottos, Lottos automaticLottos) {
+        this.manualLottos = new Lottos(manualLottos.getLottos());
+        this.automaticLottos = new Lottos(automaticLottos.getLottos());
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        this.manualLottos = manualLottos;
        this.automaticLottos = automaticLottos;
```